designsafe/static/scripts/data-depot/components/projects/publication-preview/modals/author-information-modal.component.js

```diff
@@ -13,8 +13,14 @@ class AuthorInformationModalCtrl {
         this.email = this.author.email;
         this.institution = this.author.inst;
         this.username = this.author.name;
-        this.loading = true;
-        this.getOrcid(this.username);
+        if (this.resolve.needsOrcid) {
+            this.loading = true;
```

Comment:
There shouldn't be any loading time if there is no need to get the user's info, but moving `this.loading` out of the loop will ensure it always starts "loading" on start.